modules/virt-about-top-consumers-projects-vms-nodes.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-reviewing-virtualization-overview.adoc
+
+[id="virt-about-reviewing-top-consumers-projects-vms-nodes_{context}"]
+= About top consumers for projects, virtual machines, and nodes
+
+The *Top Consumers* cards display projects, virtual machines or nodes with maximum consumption of a resource. You can select a project, a virtual machine, or a node and view the top five consumers of a specific resource.
```

Comment:
The user can select whether they would like to see the top 5 or top 10 consumers.

Reply:
Updated the content.